src/content/docs/en/guides/cms/decap-cms.mdx

```diff
@@ -8,13 +8,132 @@ i18nReady: true
 
 [Decap CMS](https://www.decapcms.org/) (formerly Netlify CMS) is an open-source, Git-based content management system.
+
+Decap allows you to take full advantage of all of Astro's features, including image optimization and content collections.
+
+Decap adds a route (typically `/admin`) to your project that will load a React app to allow authorized users to manage content directly from the deployed website. Decap will commit changes directly to your Astro project's source repository.
+
+## Installing DecapCMS
+
+There are two options for adding Decap to Astro:
+
+1. [Install Decap via a package manager](https://decapcms.org/docs/install-decap-cms/#installing-with-npm) with the following command:
+
+
+    <PackageManagerTabs>
+      <Fragment slot="npm">
+      ```shell
+      npm install decap-cms-app
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      pnpm add decap-cms-app
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      yarn add decap-cms-app
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+ 
+ 
+1. Import the package into a `<script>` tag in your page `<body>`
+
+	```html title='/admin'
+    <body>
+      <!-- Include the script that builds the page and powers Decap CMS -->
+      <script src="https://unpkg.com/decap-cms@^3.1.2/dist/decap-cms.js"></script>
+    </body>
+	```
+ 
+
+## Configuration
+
+1. Create a static admin folder at `public/admin/`
+2. Add `config.yml` to `public/admin/`: 
+    <FileTree>
+      - public
+        - admin
+          - config.yml
+    </FileTree>
+3. To add support for content collections, configure each schema in `config.yml`. The following example configures a `blog` collection, defining a `label` for each entry's frontmatter property:
+    ```yml title="/public/admin/config.yml"
+    collections:
+      - name: "blog" # Used in routes, e.g., /admin/collections/blog
+        label: "Blog" # Used in the UI
+        folder: "src/content/blog" # The path to the folder where the documents are stored
+        create: true # Allow users to create new documents in this collection
+        fields: # The fields for each document, usually in front matter
+          - { label: "Layout", name: "layout", widget: "hidden", default: "blog" }
+          - { label: "Title", name: "title", widget: "string" }
+          - { label: "Publish Date", name: "date", widget: "datetime" }
+          - { label: "Featured Image", name: "thumbnail", widget: "image" }
+          - { label: "Rating (scale of 1-5)", name: "rating", widget: "number" }
+          - { label: "Body", name: "body", widget: "markdown" }
+    ```
+
+4. Add the `admin` route for your React app. This file can be either `public/admin/index.html` alongside your `config.yml` or, if you prefer to use an Astro route, `src/pages/admin.astro`.
+
+    <FileTree>
+        - public
+          - admin
+            - config.yml
+            - index.html
+    </FileTree>
```

Comment:
```suggestion
    <FileTree>
     - public
       - admin
          - config.yml
          - index.html
    </FileTree>
```
FileTree probably doesn't want indentation to start with... let's see if this fixes it.